### DISCUSSION TITLE
Calendar View: Total hours per day

### DISCUSSION THREAD

**Discussion by gianpesto** [kimai/kimai#4276] [General]
Hi @kevinpapst ,
love the new layout, but one little feature is missing: the total booked hours per day. it would be nice if you could see the total hours you have booked in a day, maybe on the right side:

![image](https://github.com/kimai/kimai/assets/12595753/acc2c485-4221-4676-a352-9ac3270a44ac)

ciao,
gian

**Comment by empx92** [comment]
Was also the first thing our secretary said: "wow nice it works much smoother and better than before...but where are the total hours"

**Comment by empx92** [comment]
Did anyone find a workaround so far?

**Reply by kevinpapst** [reply]
Vote for it: https://github.com/kimai/kimai/issues/4292
